careers/application-process/index.html

```diff
@@ -281,13 +281,17 @@ <h3 class="u-hide-on-mobile">Asking About your Application</h3>
     <div class="block
                 block__padded-top
                 block__border-top
+                block__flush-bottom
                 content-l">
-        <section class="content-l_col content-l_col-1-2">
+        <article class="content-l_col content-l_col-1-3">
+            {% include "_template-current-openings.html" %}
+        </article>
+        <article class="content-l_col content-l_col-1-3">
             {% include "_template-working-at-cfpb.html" %}
-        </section>
-        <section class="content-l_col content-l_col-1-2">
+        </article>
```

Comment:
Sounds good to me! Looking at this I noticed it wasn't consistently one or the other either. Updated in https://github.com/cfpb/cfgov-refresh/commit/cc4a79bead7ee61d133b8881589cf554c33e22ca